tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -339,6 +340,130 @@ def test_basic_normalizer(self):
         self.assertEqual(decoded_output_diacritics, expected_output_diacritics)
 
 
+    def test_decode_asr_with_word_level_timestamps(self):
```

Comment:
Could we perhaps reduce the input size to only check the timestamps from 45->50s? This should streamline the test (isolating where the bug occurs) and make it easier to confirm the correct behaviour

Reply:
I've updated the example!